src/ResourceManager/Compute/Commands.Compute/Commands.Compute.csproj

```diff
@@ -275,8 +275,12 @@
     <Compile Include="Strategies\Compute\ComputeStrategy.cs" />
     <Compile Include="Strategies\Compute\Image.cs" />
     <Compile Include="Strategies\Compute\Images.cs" />
+    <Compile Include="Strategies\Compute\VirtualMachineScaleSetStrategy.cs" />
```

Comment:
Please update the compute changelog with a description of these changes

Reply:
fixed